### ISSUE TITLE
[FRONTEND] broken tags on bluesky posts (missaligned links, duplication)

### ISSUE THREAD

**Issue by fluffye-yay** [gabboman/wafrn#149]
Posts from bluesky may have one or more of these problems:
1. in body tags have link missaligned
2. wofrn style tags are duplicated
3. "undefined" appended

![Screenshot_20241123-063226](https://github.com/user-attachments/assets/a28a2b14-3d38-43b3-827c-c986c5b1151a)

https://app.wafrn.net/fediverse/post/28112177-fdda-4607-a747-c8f1591c6e3c

![Screenshot_20241123-063735](https://github.com/user-attachments/assets/dc15ce9a-ea8b-4a5b-8ad6-a11c386e4410)

https://app.wafrn.net/fediverse/post/fb17b4f4-ffc1-4c0f-b5c5-c61f590a55b6

**Comment by gabboman** [comment]
yep, it happens on some old posts. in THEORY should be fixed but I will keep an eye on it
